lib/bolt/cli.rb

```diff
@@ -511,7 +511,7 @@ def execute(options)
           options[:task_options]['nodes'] = options[:nodes].join(',')
         end
         executor = Bolt::Executor.new(config, options[:noop], true)
-        result = pal.run_plan(options[:object], options[:task_options], executor, inventory)
+        result = pal.run_plan(options[:object], options[:task_options], executor, inventory, puppetdb_client)
```

Comment:
@adreyer should we catch and log debug if constructing puppetdb_client fails? Or wrap it in a function that initializes it the first time it's used?

Reply:
Made a wrapper to delay construction until we try to use the client.

Reply:
I don't think we want to always fail if puppetdb isn't configured. This delays errors until use, which seems like the only viable UX.

Reply:
agreed